src/calculateRentalCost.js

```diff
@@ -28,7 +28,25 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const BASE_RENT_COST = 40;
+
+  const SHORT_RENT_TERM = 3;
+  const SHORT_RENT_DISCOUNT = 20;
+
+  const LONG_RENT_TERM = 7;
+  const LONG_RENT_DISCOUNT = 50;
+
+  const rentCost = days * BASE_RENT_COST;
```

Comment:
const values should be named with uppercase?)

Reply:
> const values should be named with uppercase?)

Only constant variables. In this case, the variable depends on the `days` parameter, so it's just a variable.